content/docs/user-guide/basic-concepts/dvc-project.md

```diff
@@ -12,7 +12,36 @@ match:
     repositories
   ]
 tooltip: >-
-  Initialized by running `dvc init` in the **workspace** (typically a Git
-  repository). It will contain the `.dvc/` directory, as well as `dvc.yaml` and
-  `.dvc` files created with commands such as `dvc add` or `dvc run`.
+  A DVC project is a directory initialized by `dvc init` that contains a `.dvc`
+  directory. It contains all <abbr>metafiles</abbr> related to project and also
+  includes the underlying <abbr>cache</abbr>.
+  [📖](/doc/user-guide/basic-concepts/dvc-project)
 ---
+
+# DVC Project
+
+A DVC project (or DVC repository) is a directory initialized by `dvc init`. It
```

Comment:
> A DVC project (or DVC repository) is a directory

Not sure we want to say it's a project is a directory. That increases the overlap with the workspace concept. Please refer to the discussion we had earlier in https://github.com/iterative/dvc.org/issues/1127#issuecomment-773630307